Extensions/PlatformBehavior/tests/FlatFloorPlatformer.spec.js

```diff
@@ -404,7 +404,9 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
         expect(object.getY()).to.be(-30); // -30 = -10 (platform y) + -20 (object height)
 
         // Make the platform under the character feet smaller.
-        object.setCustomWidthAndHeight(object.getWidth(), 9);
+        runtimeScene.renderAndStepWithEventsFunction(1000 / 60, () => {
+          object.setCustomWidthAndHeight(object.getWidth(), 9);
+        });
         runtimeScene.renderAndStep(1000 / 60);
         expect(object.getBehavior('auto1').isFalling()).to.be(false);
```

Comment:
I added a step because the change of state will only happens after the 2nd step as the platform is now moving after `doStepPreEvents` as it should.
I didn't do it on the other tests because they are looping so event if the 1st iteration can't really change of state, it's not a big deal because other step follows.